test/Microsoft.AspNetCore.Mvc.FunctionalTests/FormatFilterSampleTest.cs

```diff
@@ -42,8 +43,15 @@ public FormatFilterSampleTest(MvcTestFixture<FormatFilterSample.Web.Startup> fix
         [Fact]
         public async Task FormatFilter_ExtensionInRequest_Optional()
         {
-            // Arrange & Act
-            var response = await Client.GetAsync("http://localhost/FormatFilter/GetProduct.json");
+            // Arrange
+            var request = new HttpRequestMessage();
```

Comment:
Be more explicit and shorten the setup to match most other `HttpRequestMessage`s used in tests: `new HttpRequestMessage(HttpMethod.Get, "http://localhost/FormatFilter/GetProduct.json")`.

Then remove [line 51](https://github.com/aspnet/Mvc/pull/4034/files#diff-59701ed5c13c8176b640aaa41f872c67R51).